src/clients/inst_add-on.rb

```diff
@@ -5,6 +5,12 @@
 # Summary:	Select add-on products for installation
 # Authors:	Jiri Srain <jsrain@suse.de>
 #
+
+# NOTE: this client should not be called from other clients directly
```

Comment:
ok, just maybe move note to class so it is also in yardoc and also would be nice to use `@note` as yardoc tag